sdk/core/azure-core/azure/core/pipeline/transport/_base.py

```diff
@@ -81,6 +74,7 @@
 HTTPResponseType = TypeVar("HTTPResponseType")
 HTTPRequestType = TypeVar("HTTPRequestType")
 PipelineType = TypeVar("PipelineType")
+Message = TypeVar("Message")
```

Comment:
Seems unused

Reply:
It is used on https://github.com/Azure/azure-sdk-for-python/blob/core_lazy_load_aiohttp/sdk/core/azure-core/azure/core/pipeline/transport/_base.py#L510